src/Docker/docker-compose.yml

```diff
@@ -88,6 +91,8 @@ services:
       BASE_PATH: /analytics
       CLIENT_IP_HEADER : "x-original-ip"
     restart: always
+    ports:
+      - '3003:3000'
```

Comment:
Is this change needed? I'm always nervous about unnecessary changes in this file :-)